src/lib/y2partitioner/widgets/disk_table.rb

```diff
@@ -69,22 +71,16 @@ def header
         ]
       end
 
+      HELP_FIELDS = [:device, :size, :format, :encrypted, :type, :fs_type,
+                     :label, :mount_point, :start_cyl, :end_cyl].freeze
```

Comment:
as I write it is just stupid copy and paste of old storage approach with fields. I add also TODO there for this, but now I am a bit under time pressure and I hope we do some real UX testing of helps and write to widgets what is not clear for users.